iphone/Maps/Maps.xcodeproj/project.pbxproj

```diff
@@ -6155,12 +6162,18 @@
 				ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
 				BUNDLE_IDENTIFIER = app.omaps.release;
 				CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES = YES;
+				CLANG_UNDEFINED_BEHAVIOR_SANITIZER_NULLABILITY = NO;
```

Comment:
Are you sure about this?

Reply:
https://clang.llvm.org/docs/analyzer/developer-docs/nullability.html

Clang supports special `nonnull` attribute for pointers. Some systems libraries use this attribute. Our code is not ready for that.